ab3d2_source/c/main.c

```diff
@@ -0,0 +1,38 @@
+#include "system.h"
+#include "screen.h"
+
+#include <graphics/modeid.h>
+#include <proto/cybergraphics.h>
+
+#include <stdio.h>
+
+extern void startup(void);
+
+const long __nocommandline=1;
+
+int main(int argc, char *argv[])
+{
+    int rval = 10;
+    if (!sys_OpenLibs())
```

Comment:
Nitpick: Now that sys_OpenLibs() and sys_CloseLibs() are called externally from the system module, if we are being strict with conventions set before then these should be renamed to Sys_OpenLibs and Sys_CloseLibs.

Reply:
will do